specification/containerinstance/resource-manager/Microsoft.ContainerInstance/2017-12-01-preview/containerInstance.json

```diff
@@ -930,6 +1047,13 @@
       "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call.",
       "x-ms-parameter-location": "client"
     },
+    "LocationParameter": {
+      "name": "location",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The identifier for the physical azure location."
```

Comment:
If we don't specify  "x-ms-parameter-location": "method" then this become property in the client like subscription property, but seems LocationParameter is only used in the _List method hence you might want to apply this extension so that this become a parameter of _List method instead of client level global property